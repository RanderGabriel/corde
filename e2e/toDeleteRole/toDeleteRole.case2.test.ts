import Utils from "../testUtils";
import { runCLI } from "../cliRunner";

it("should fail", async () => {
  const command = Utils.buildCommandWithConfigPath("toDeleteRole", "bot_case2.test.ts");
  const results = await cli.exec(command);
  expect(results.statusCode).toEqual(1);
  expect(results.stdout).toMatchSnapshot();
});
