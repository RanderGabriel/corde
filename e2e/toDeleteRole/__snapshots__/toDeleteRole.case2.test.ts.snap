// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should fail 1`] = `
"
 FAIL   e2e\\\\toDeleteRole\\\\__cordeTest__\\\\bot_case2.test.ts   100ms
   x should fail in delete a role   100ms
    
    timeout: role 825879364813062185 wasn't deleted in the expected time (5000)

    at toDeleteRole e2e\\\\toDeleteRole\\\\__cordeTest__\\\\bot_case2.test.ts:8:45
    at lib\\\\src\\\\clausures\\\\test.js:30:15
    at lib\\\\src\\\\utils\\\\executePromiseWithTimeout.js:53:5

Test Files: 1 failed. 1 total
Tests:      1 failed. 1 total
Time:       100ms
"
`;
